pkg/kubectl/cmd/util/openapi/validation/validation.go

```diff
@@ -61,15 +61,15 @@ func (v *SchemaValidation) validateList(object interface{}) []error {
 		return []error{errors.New("invalid object to validate")}
 	}
 
-	errs := []error{}
+	allErrors := []error{}
 	for _, item := range fields["items"].([]interface{}) {
-		if gvk, err := getObjectKind(item); err != nil {
-			errs = append(errs, err)
+		if gvk, errs := getObjectKind(item); allErrors != nil {
```

Comment:
This should be `errs != nil`

Reply:
It looks like the code that you're changing is not validated by any unit-test. Would you mind (in this PR or in a in follow-up PR) improve the unit-test thoroughness?. For example, I could have missed that mistake, and we should be confident that the tests are going to catch it.

Reply:
@apelisse hi, yup agree and absolutely I would work on it as a follow up PR(s). I actually wanted to do it and tried playing with validation_test. So I wanted to touch base with you on it too. Thanks!!